docs/usage.md

```diff
@@ -568,22 +671,32 @@ If you prefer, you can specify the full path to your reference genome when you r
 --intervals '[path to the intervals file]'
 ```
 
-### --knownIndels
+### --known_indels
 
 If you prefer, you can specify the full path to your reference genome when you run the pipeline:
 
 ```bash
---knownIndels '[path to the knownIndels file]'
+--known_indels '[path to the knownIndels file]'
 ```
 
-### --knownIndelsIndex
+### --knownIndels
+
+> :warning: This params has been removed.
+> Please check: [`--known_indels`](#--known_indels)
+
+### --known_indels_index
 
 If you prefer, you can specify the full path to your reference genome when you run the pipeline:
 
 ```bash
---knownIndelsIndex '[path to the knownIndels index]'
+--known_indels_index '[path to the knownIndels index]'
 ```
 
+### --knownIndelsIndex
+
+> :warning: This params has been removed.
+> Please check: [`--known_indels_index`](#--known_indels_index)
+
 ### --pon
 
 When a panel of normals [PON](https://gatkforums.broadinstitute.org/gatk/discussion/24057/how-to-call-somatic-mutations-using-gatk4-mutect2#latest) is defined, you will get filtered somatic calls as a result.
```

Comment:
```suggestion
When a panel of normals [PON](https://gatkforums.broadinstitute.org/gatk/discussion/24057/how-to-call-somatic-mutations-using-gatk4-mutect2#latest) is defined, you will get better filtered somatic calls as a result.
```

Reply:
What do you think about that phrase?

Reply:
Sounds good! How do you want me to deal with your changes? Commit suggestion directly? Or how can I do it so it's easiest for you?

Reply:
Yes, commit suggestion directly is good with me